Godeps/_workspace/src/github.com/Clever/pathio/Makefile

```diff
@@ -0,0 +1,29 @@
+SHELL := /bin/bash
+PKG = github.com/Clever/pathio
+PKGS := $(PKG)
+READMES = $(addsuffix /README.md, $(PKGS))
```

Comment:
This is the makefile of clever/pathio which is an dependency. I assume that I shouldn't make any changes here.